server/main.lua

```diff
@@ -2,12 +2,6 @@ RegisterNetEvent('tackle:server:TacklePlayer', function(playerId)
     TriggerClientEvent('tackle:client:GetTackled', playerId)
 end)
 
-lib.addCommand('id', {
-    help = 'Check Your ID #',
-}, function(source)
-    TriggerClientEvent('QBCore:Notify', source,  'ID: '..source)
-end)
-
```

Comment:
What does this have to do with qbx_socreboard?

Reply:
Figured I'd put it in the scoreboard where you see your ID anyway instead of just leaving it in smallresources. If you know a better place to put it, we can move it.